README.md

```diff
@@ -466,40 +466,45 @@ PropertyTestEntity::PropertyTestEntity()
+To store geometric data, SEMPR relies on [GEOS](http://geos.osgeo.org), which implements a standard proposed by the _Open Geospatial Consortium (OGC)_. The class hierarchy of `geos::geom::Geometry` is mirrored in entities with the `Geometry`-base-class. The entity-classes simply wrap a corresponding geometry: `entity::Point` contains an `geos::geom::Point*` etc.
 
 #### Spatial reference systems / coordinate systems
 Geometric data is not always stored in the same global reference system. In some cases it might be even beneficial to create local coordinate systems that are related to each other: If a parent system is changed, the children stay in relative position/orientation to their parent (and change on a more global scale). In order to support such chains and different reference systems -- even geographic ones -- the following distinctions have been made, and classes have been implemented.
+
 ##### Global references
-Global reference systems inherit from the `GlobalCS`-class and are implicitly linked to each other, with the earth as a common root. Since "earth" is not modeled explicitely, the global references are also referred to as "root reference systems". Currently, they are supported through two classes: `GeographicCS` models geographic coordinate systems which are based on latitude and longitude coordinates. They do cannot be chained with other reference systems: How would an affine transformation relative to "WGS84" make any sense?
+Global reference systems inherit from the `GlobalCS`-class and are implicitly linked to each other, with the earth as a common root. Since "earth" is not modeled explicitely, the global references are also referred to as "root reference systems". Currently, they are supported through three classes: `GeodeticCS` models geographic coordinate systems which are based on latitude and longitude coordinates on a WGS84 ellipsoid with Greenwich as meridian. 
+They do cannot be chained with other reference systems: How would an affine transformation relative to "WGS84" make any sense?
+
+In most cases you will want to represent data that is not directly aquired in (lon lat) but rather (x y z) coordinates, but which may be linked to a specific geographic location. This is what a `ProjectionCS` is used for: It defines a reference system projected to/from an underlying geographic one. Currently implemented are UTM, UPS and MGRS (also known as UTMREF) projections. The latter can be attached to any (lat lon) coordinate and defines a plane that touches the globe at that location. Since at this point the coordinates of any geometry attached to the coordinate system is interpreted as (x y z), it is possible to attach local reference systems to it.
```

Comment:
> The latter can be attached to any (lat lon) coordinate and defines a plane that touches the globe at that location.

In the previous version, "the latter" was the equirectangular coordinate system. Since the sentenced changed, "the latter" is now MGRS/UTMREF, which actually can **not** be attached to just any lat long. I guess you should reference the `LocalTangentPlaneCS` here or restructure the paragraph.